{"name": "Логин несуществующего пользователя: 404 и сообщение об ошибке", "status": "passed", "steps": [{"name": "Пытаемся залогиниться с несуществующими учетными данными", "status": "passed", "attachments": [{"name": "response.json", "source": "4d6b05dc-48c4-49af-a47a-af6e539b0a90-attachment.json", "type": "application/json"}], "start": 1761051189473, "stop": 1761051189678}, {"name": "Проверяем 404 и текст ошибки", "status": "passed", "start": 1761051189678, "stop": 1761051189678}], "start": 1761051189473, "stop": 1761051189678, "uuid": "a6b6b428-8af7-4ee6-bd69-b5f26b4f17a8", "historyId": "e4e1a5d5dd68c2cbdeede2f5a294d6e4", "testCaseId": "e4e1a5d5dd68c2cbdeede2f5a294d6e4", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_nonexistent_user_returns_error", "labels": [{"name": "epic", "value": "Scooter API"}, {"name": "story", "value": "Login courier"}, {"name": "feature", "value": "Courier"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "n00office895.corp.tander.ru"}, {"name": "thread", "value": "41354-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}